snode-src/duckcoin.go

```diff
@@ -156,6 +157,7 @@ func main() {
 	mine(numOfBlocks, data, receiver, amount)
 }
 
+// mine mines a given number of blocks with the given data, receiver, and amount.
```

Comment:
The info "mine mines a given number of blocks with the given data, receiver, and amount" is already given by the actual code. We'll need to add more info for the comment to be useful.
```suggestion
// mine mines numOfBlocks blocks, with the arbitrary data field set to data. It also takes in the receiver's address and amount to send in each block, if the block should contain a transaction.
```